extensions/default/src/DicomJSONDataSource/index.js

```diff
@@ -258,19 +262,19 @@ function createDicomJSONApi(dicomJsonConfig) {
 
```

Comment:
On the line just above const images = displaySet.images, do you mean to fetch images, or instances?  They aren't necessarily the same as the instances is defined for other display set types, while images is only defined for image containing types.  Also, instances can include the overall sop instance, without frame number, while images are supposed to be individual images with frame number.

Reply:
I wanted all instances from a multiframe json file instead of just one instance representing the multiframe. My use case had a json file with a single multiframe series but It had individual URL attributes per frame.

Reply:
Code updated!